cmake/modules/Libbacktrace.cmake

```diff
@@ -0,0 +1,18 @@
+include(ExternalProject)
+
+ExternalProject_Add(project_libbacktrace
```

Comment:
Thanks for the heads up. I've fixed it so that libbacktrace.a depends on its sources correctly. Note that there is a bug with cmake and ninja where dependencies aren't tracked correctly, so you'll have to build twice with ninja when updating libbacktrace. But with make everything works fine.